wolfcrypt/test/test.c

```diff
@@ -2080,7 +2080,7 @@ int poly1305_test(void)
     byte     tag[16];
     Poly1305 enc;
 
-    const byte msg[] =
+    static const byte msg[] =
```

Comment:
Is there a reason these need to remain in memory after the function has exited specific to MDK5?

comment refers to lines:
2083, 2092, 2098, 2106, 2131, 2138, ..., and 2170

Having a difficult time justifying the requirement for 'static'. Please explain.